src/main/java/com/celements/model/access/DefaultModelAccessFacade.java

```diff
@@ -359,9 +358,10 @@ public BaseObject getXObject(XWikiDocument doc, DocumentReference classRef, Stri
   }
 
   private void checkNotTranslation(XWikiDocument doc) {
-    String msg = "Trying to access XObjects on translation '{0}' of doc '{1}";
-    checkState(!isTranslation(doc), MessageFormat.format(msg, doc.getLanguage(),
-        doc.getDocumentReference()));
+    if (isTranslation(doc)) {
```

Comment:
keep using MessageFormat.format but fix the wrong usage. I expect that the single quotes cause the problem. Extract the Error-Message to a constant and check in a simple unit test that the replacement with MessageFormat is working.